index.html

```diff
@@ -91,7 +91,7 @@ <h2 class="head-text">Software Developer</h2>
   </section>
   <section class="multi-post">
     <div class="header work">
-      <h1> MY RECENT WORK</h1>
+      <h2> MY RECENT WORK</h2>
```

Comment:
- [ ] Colour contrast
 - Your project has some failed contrast color pairs. currently, the colors you used for buttons in your projects section are failing use this [Link](https://www.aditus.io/)  to see any fix you can do.